Source/WebKit/GPUProcess/graphics/WebGPU/RemoteGPU.h

```diff
@@ -121,6 +126,7 @@ class RemoteGPU final : public CanMakeWeakPtr<RemoteGPU>, public IPC::StreamServ
 
 
     void requestAdapter(const WebGPU::RequestAdapterOptions&, WebGPUIdentifier, CompletionHandler<void(std::optional<RemoteGPURequestAdapterResponse>&&)>&&);
+    void addMeshRequest(const DDModel::DDMeshDescriptor&, WebKit::DDModelIdentifier);
```

Comment:
We intentionally use RemoteGPU for two reasons:

(1) rendering the models requires access to a MTLDevice
(2) we want the rendering to occur on the same thread as other GPU tasks so <model> can eventually take advantage of WGSL shaders